src/olympia/addons/views.py

```diff
@@ -183,6 +187,7 @@ class AddonViewSet(RetrieveModelMixin, GenericViewSet):
             AllowUnlistedViewerOrReviewer,
         ),
     ]
+    authentication_classes = [JWTKeyAuthentication, WebTokenAuthentication]
```

Comment:
It's probably safe to do (ultimately this doesn't let you do that much), but this makes the whole API available with external auth and should be documented.

Reply:
Documented in the API docs?  Or elsewhere

Reply:
In the API docs. For addons (and many others), we currently say `The only authentication method available at the moment is the internal one.` and this wouldn't be true anymore after these changes.

Reply:
The only thing needed is to remove that whole sentence for addons API I think.